pixart.js

```diff
@@ -1 +1,81 @@
+$(document).ready(function() {
+  color = $('.brush').css('background');
+  prior = [color];
 
+  //create swatches
+  for (i = 1; i <= 3; i++) {
+    $('.brush').append('<div></div>');
+    $('div').eq(i + 1).attr('class', 'swatch');
+    $('div').eq(i + 1).attr('id', 'swatch' + i);
+  }
+
+  $('.swatch').css('height', '30px');
+  $('.swatch').css('width', '30px');
+  $('.swatch').css('margin', '5px 5px 5px 130px');
+
+  //create canvas
+  for(i=1; i<=8000; i++){
+    var divNum = i+4;
+    $('body').append('<div></div>');
+    $('div').eq(divNum).attr('class', 'square');
+  }
+
+  $('.square').css('height', '10px');
+  $('.square').css('width', '10px');
+  $('.square').css('margin', '0');
```

Comment:
Thanks, John! I knew there had to be some shortcut, but I couldn't figure
one out.

On Thu, Oct 29, 2015 at 2:08 PM, John Master <notifications@github.com>
wrote:

> In pixart.js
> <https://github.com/ga-dc/pixart_js/pull/125#discussion_r43422564>:
>
> > +  }
> > +
> > +  $('.swatch').css('height', '30px');
> > +  $('.swatch').css('width', '30px');
> > +  $('.swatch').css('margin', '5px 5px 5px 130px');
> > +
> > +  //create canvas
> > +  for(i=1; i<=8000; i++){
> > +    var divNum = i+4;
> > +    $('body').append('<div></div>');
> > +    $('div').eq(divNum).attr('class', 'square');
> > +  }
> > +
> > +  $('.square').css('height', '10px');
> > +  $('.square').css('width', '10px');
> > +  $('.square').css('margin', '0');
>
> Can also give .css an object of key value pairs so:
>
> $('.square').css({
>   height: '10px',
>   width: '10px',
>   margin: '5px 5px 5px 130px'
> })
>
> would have the same effect
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-dc/pixart_js/pull/125/files#r43422564>.
>